docs/can-guides/upgrade/using-codemods.md

```diff
@@ -91,10 +91,10 @@ Runs all the `can-migrate` transforms for upgrading to CanJS 4.0 on the files th
 can-migrate '**/*.js' --can-version 4 --apply
 ```
 
-Runs the `can-stache/route-helpers` transform on the files that match the `**/*.js` glob:
+Runs the `can-stache/route-helpers` transform on the files that match the `**/*.stache` glob:
 
 ```bash
-can-migrate '**/*.js' --transform can-stache/route-helpers.js --apply
+can-migrate '**/*.stache' --transform can-stache/route-helpers.js --apply
```

Comment:
I think this one’s confusing because it applies to both JS & stache. I’d leave it as-is or change the example to something that only applies to one or the other.